docs/source/docker_files.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../../docker/README.md
+```
```

Comment:
This is a kind of duplication I think. An entire file only to call another one. Couldn't we include the README directly in the source index and not use md for the moment? This file is the one thing that would justify adding the package but the file itself is not necessary.

Reply:
Ok for including the file then, but using md in a single file does not justify adding the package to support md in the documentation. Unless we start using md more frequently, adding the package is not well motivated for me.